data/sqlutil/dynamic_frame.go

```diff
@@ -10,13 +10,18 @@ import (
 
 const STRING = "string"
 
-func isDynamic(converters []Converter) bool {
+// reoveDynamicConverter filters out the dynamic converter.  It is not a valid converter.
+func removeDynamicConverter(converters []Converter) (bool, []Converter) {
+	var filtered []Converter
+	var isDynamic bool
 	for _, conv := range converters {
-		if conv.Dynamic {
-			return true
+		if !conv.Dynamic {
+			filtered = append(filtered, conv)
+			continue
 		}
+		isDynamic = true
```

Comment:
a little bit more readable IMO if this is:

```suggestion
		if conv.Dynamic {
			isDynamic = true
		} else {
			filtered = append(filtered, conv)
		}
```

Reply:
@andresmgot i think you are right.  but do you think a continue in the if condition would be better than else?

Reply:
mostly the same, both are good